licenses.json

```diff
@@ -2,5 +2,6 @@
   "GPL-2.0-or-later": "GPLv2+",
   "GPL-3.0": "GPLv3",
   "GPL-3.0+": "GPLv3+",
-  "GPLv3-or-later": "GPLv3+"
+  "GPLv3-or-later": "GPLv3+",
```

Comment:
AFAIK this is not a valid SPDX license code. Should we keep it out of the cherry pick commit?

Reply:
GPL-3.0-or-later is valid, and that's what I added. GPLv3-or-later is not, but that was here before.

Reply:
Yes, I later realized this was the cherry pick PR.